vignettes/tech-dm-keyed.Rmd

```diff
@@ -0,0 +1,216 @@
+---
+title: "Manipulating individual tables"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Technical: Manipulating individual tables}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+``````{r setup, include = FALSE}
+source("setup/setup.R")
+``````
+
+This vignette deals with situations where you want to transform tables of your `dm` and then update an existing table or add a new table to the `dm`.
+There are two straightforward approaches:
+
+1. individually subsetting the tables relevant to the calculation, perform the necessary transformations, add the result to the `dm` (or replace an existing table), and establish the key relations.
+1. do all this within the `dm` object by zooming to a table and manipulating it while maintaining the key relations whenever possible.
+
+We will explore the first approach here. For the second approach, see `vignette("tech-dm-zoom")`.
+
+## Enabling {dplyr}-workflow within a `dm`
+
+Subsetting a table of a `dm` object produces an object of class `dm_keyed_tbl`.
+
+{dm} provides methods for many of the {dplyr}-verbs for a `dm_keyed_tbl` which behave the way you are used to, affecting only the table and leaving the rest of the `dm` untouched.
```

Comment:
is the use of `{}` a convention in all docs? if so we should add it to the contributing guide.

Reply:
how to know which verbs work?